packages/@okta/vuepress-site/code/vue/okta_vue_sign-in_widget/index.md

```diff
@@ -0,0 +1,358 @@
+| -------------------  | --------------------------------------------------- |
+| App Name             | OpenId Connect App *(must be unique)*               |
+| Login redirect URIs  | http://localhost:8080/login/callback             |
+| Logout redirect URIs | http://localhost:8080                         |
+| Allowed grant types  | Authorization Code                                  |
+
+> **Note:** CORS is automatically enabled for the granted login redirect URIs.
+
+## Create a Vue App
+
+To quickly create a Vue app, we recommend the Vue CLI.
+
+```bash
+npm install -g @vue/cli
+vue create okta-app
+# Manually select features: choose defaults + Router
+# Choose history mode for router
+cd okta-app
+```
+
+If you need more information, see [the Vue CLI installation guide](https://cli.vuejs.org/guide/installation.html).
+
+## Install Dependencies
+
+A simple way to add authentication into a Vue app is using the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) library. You can install it via `npm`:
+
+```bash
+cd okta-app
+npm install @okta/okta-signin-widget
+```
+
+You'll also need `@okta/okta-vue` for route protection:
+
+```bash
+npm install @okta/okta-vue
+```
+
+## Create a Widget Wrapper
+
+To provide a fully featured and customizable login experience, the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) is available to handle User Lifecycle operations, MFA, and more. To render the Sign-In Widget in Vue, you must create a wrapper that allows us to treat it as a Vue component.
+
+Create a `src/components/Login.vue` file:
+
+```html
+<template>
+  <div class="login">
+    <div id="okta-signin-container"></div>
+  </div>
+</template>
+
+<script>
+import OktaSignIn from '@okta/okta-signin-widget'
+import '@okta/okta-signin-widget/dist/css/okta-sign-in.min.css'
+
+export default {
+  name: 'Login',
+  mounted: function () {
+    this.$nextTick(function () {
+      this.widget = new OktaSignIn({
+        baseUrl: 'https://{yourOktaDomain}',
+        clientId: '{clientId}',
+        redirectUri: 'http://localhost:8080/login/callback',
+        authParams: {
+          pkce: true,
+          issuer: 'https://{yourOktaDomain}/oauth2/default',
+          display: 'page',
+          scopes: ['openid', 'profile', 'email']
+        }
+      })
+      this.widget.renderEl(
+        { el: '#okta-signin-container' },
+        () => {
+          /**
+           * In this flow, the success handler will not be called because
+           * there's a redirect to the Okta org for the authentication workflow.
+           */
+        },
+        (err) => {
+          throw err
+        }
+      )
+    })
+  },
+  destroyed () {
+    // Remove the widget from the DOM on path change
+    this.widget.remove()
+  }
+}
+</script>
+```
+
+## Create Routes
+
+Some routes require authentication in order to render. Defining those routes is easy using Vue Router and `@okta/okta-vue`. Lets take a look at what routes are needed for this example:
```

Comment:
```suggestion
Some routes require authentication in order to render. Defining those routes is easy using Vue Router and `@okta/okta-vue`. Let's take a look at what routes are needed for this example:
```

Reply:
This matches what React has. Should I change React's guide to use "Let's" instead of "Lets" as well?